doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -137,13 +147,25 @@ The data path and packet format of the DPU-to-DPU probe will be the same as the
 
 In `DPU-Scope-DPU-Driven` setup, the HA state machine will be managed by DPU itself, and `hamgrd` will not drive the HA state machine, but only be used for generating the configurations for NPU and DPU, as well as collecting the telemetry and report in the context of HA whenever is needed.
 
-### 7.1. HA state
+### 7.1. HA states
 
 Since DPU will be driving the HA state machine transition, any HA state change will needs to be reported, otherwise the SDN controller will not be able to know the current HA state of the DPU.
 
-With this setup, all HA states we defined for the ENI-level HA setup can be used to map the undering states, not only `Dead`, `Active`, `Standby` and `Standalone`.
-
-For details on the states and the expected behavior, please refer to the [HA states defined in main HA design doc](./smart-switch-ha-hld.md#71-ha-state-definition-and-behavior).
+The DPU driven state machine will be in one of the following operational HA states at any given point
+
+ | State |    Description                       |
+ | ----- | ------------------------------------ |
+ | Dead | Initial state not participating in HA |
+ | Connecting | Trying to connect to its HA pair |
+ | Connected | Connection successful, bulk sync in progress |
+ | InitializingToStandalone | Could not connect to pair, waiting for Activation from SDN controller to go to Standalone. Dormant state since BFD disabled hence no traffic. |
+ | InitializingToActive | Bulk sync successful, waiting for Activation from SDN controller to go to Active. Dormant state since BFD disabled hence no traffic. |
+ | InitializingToStandby | Bulk sync successful, waiting for Activation from SDN controller to go to Standby. Dormant state since BFD disabled hence no traffic. |
+ | Standalone | Activation done, forwarding traffic |
+ | Active |  Activation done, forwarding traffic and syncing flows to pair |
+ | Standby |  Activation done, forwarding traffic and syncing flows from pair |
```

Comment:
I think it probably not good to simply say "Standby will forward the traffic" here. This will be very counter-intuitive. We need some background context here.

Reply:
copy-paste error. fixed